### PR TITLE
Add a code of conduct.

### PR DESCRIPTION
**Pull request by kalaspuffar**
Hi @bertfrees, @PaulRambags, and @joeha480 

What do you think about this code of conduct?
Could this be added to all repositories?

Best regards
Daniel

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at daniel.persson@textalk.se. All
```
Comment (PaulRambags) [inline CODE_OF_CONDUCT.md:58]: This is a personal e-mail. That means, when Daniel stops working for this project, all code must be changed. Would it not be better to have a more general e-mail address, for instance, info@textalk.se?

**Review by PaulRambags — COMMENTED** [review]
I have added one comment

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (kalaspuffar) [reply CODE_OF_CONDUCT.md:58]: Hi @PaulRambags 

Good suggestion, but I think it's better to keep sensitive issues to a small team or on a personal level. If you feel abused you might not want to email a address that reaches 10 people inside Textalk.

It's safer if one or perhaps two people handle these issues. If you feel it's inappropriate that I take that role or if you want a less distinct email we can also change it to something like abuse@textalk.se

I'm ready to handle these kinds of issues and have done before. And we can always change this in the future if I need to go on from Textalk.

What do you think @joeha480 and @bertfrees ?

Best regards
Daniel

**Comment by joeha480** [comment]
I actually already had a Code of Conduct in mind. I just forgot to add it (didn't think I needed to, to be honest). Anyway, it's fixed now.

**Comment by bertfrees** [comment]
LOL at Joel's reaction. A bit harsh, but I guess you want to make a statement.

To be really honest I hadn't heard from CoC's before, and I don't really see the added value of it either (in our case). I mean of course there's nothing wrong with what's in the text, but isn't it all very logical? Do you need a text to remind you that you can't be a jerk? To me it sounds a bit too politically correct. OK I can understand that if a project has a history of incidents with insulting language or something like that... but we're civilized people.

**Pull request closed without merging**